examples/client/Locomotion/src/Components/BsPages/index.tsx

```diff
@@ -344,13 +349,15 @@ export const ConfirmPickupTime = (props: any) => {
       </RoundedButton>
       <DatePickerPoppup
         testID="datePicker"
-        textColor={getTextColorForTheme()}
+        textColor="black"
```

Comment:
should it be hardcoded? what about dark mode?

Reply:
we don't have different colors for all new ride process when on dark mode